src/plugins/overlay/overlay-en.hbs

```diff
@@ -250,6 +261,63 @@ wb.doc.on( "click vclick", "#overlay-open-btn", function( event ) {
 &lt;/section&gt;</code></pre>
 		</details>
 	</section>
+
+	<section>
+		<h3>Fix header and footer to overlays</h3>
+		<p>Add the CSS <code>wb-popup-flex</code> to the overlay dialog to enable the fix header and footer so only the body section will scroll up and down.</p>
+		<p><a href="#br-screen-flex" aria-controls="br-screen-flex" class="overlay-lnk" role="button">Bottom-right overlay with fix header and footer</a></p>
+		<details class="mrgn-bttm-lg">
+			<summary>View code</summary>
+			<pre><code>&lt;p&gt;&lt;a href=&quot;#br-screen-flex&quot; aria-controls=&quot;br-screen-flex&quot; class=&quot;overlay-lnk&quot; role=&quot;button&quot;&gt;Bottom-right overlay with fix header and footer&lt;/a&gt;&lt;/p&gt;
+			&lt;section id=&quot;br-screen-flex&quot; class=&quot;wb-overlay modal-content overlay-def wb-popup-br wb-popup-flex&quot;&gt;
+			&lt;header class=&quot;modal-header&quot;&gt;
+			&lt;h2 class=&quot;modal-title&quot;&gt;Bottom-right overlay with fix header and footer&lt;/h2&gt;
+			&lt;/header&gt;
+			&lt;div class=&quot;modal-body&quot;&gt;
+			...
+			&lt;/div&gt;
+		&lt;/section&gt;</code></pre>
+		</details>
+	</section>
+
+	<section>
+		<h3>Remove default footer or disable the close event from overlays</h3>
+		<p>Add <code>data-wb-overlay</code> to customize footer and or the close event.</p>
+		<ul>
+			<li><code>{ &quot;noFooter&quot;: true }</code> this will remove the default footer from overlays so a customized footer can be added.</li>
+			<li>><code>{ &quot;noClose&quot;: true }</code> this will prevent overlays to close on mouse right-click event or when pressing the escape key.</li>
```

Comment:
Small typo here:
```suggestion
			<li><code>{ &quot;noClose&quot;: true }</code> this will prevent overlays to close on mouse right-click event or when pressing the escape key.</li>
```